packages/blockchain-extension/resources/tutorials/developer-tutorials/privateData.md

```diff
@@ -90,11 +88,13 @@ The `verifyMyPrivateAsset` transaction can be used to verify if a given set of p
 
     public async verifyMyPrivateAsset(ctx: Context, myPrivateAssetId:   string, objectToVerify: MyPrivateAsset): Promise<boolean>
 
-This transaction requires two arguments – `myPrivateAssetId` and the `objectToVerify` (made up of a key/value pair). The `myPrivateAssetId` is simply the key that was used when saving the private data (supplied when passing in transient data) to the Private Data Collection. The object key and value will appear in the format `{"key":value"}`. For example, if Alice stores some private data `{"myPrivateValue":"50"}` in her Private Data Collection which only she has access to but wants to prove to Bob that it exists, Bob would use the verify transaction to confirm whether Alice is telling the truth or not. Bob can now submit the `verifyPrivateAsset()` transaction with the arguments: `asset ID` which is on the public ledger as `myPrivateAssetID` and the object `{"myPrivateValue":"50"}`. If Alice provided the correct data, the transaction should return true as the hash provided matches the hash stored on the public ledger. We will look at this in more detail later.
+This transaction requires three arguments – `mspid`, `myPrivateAssetId` and the `objectToVerify` (made up of a key/value pair). The `mspid` is the ID of the organization that you want to verify against, for example if Bob wants to verify Alice's data then he would input Alice's MSPID. The `myPrivateAssetId` is simply the key that was used when saving the private data (supplied when passing in transient data) to the Private Data Collection. The object key and value will appear in the format `{"key":value"}`. For example, if Alice stores some private data `{"myPrivateValue":"50"}` in her Private Data Collection, identifiable by the MSPID `AliceOrg1MSP`, which only she has access to but wants to prove to Bob that it exists, Bob would use the verify transaction to confirm whether Alice is telling the truth or not. Bob can now submit the `verifyPrivateAsset()` transaction with the arguments: `mspid` Alice's is `AliceOrg1MSP`, `asset ID` which is on the public ledger as `myPrivateAssetID` and the object `{"myPrivateValue":"50"}`. If Alice provided the correct data, the transaction should return true as the hash provided matches the hash stored on the public ledger. We will look at this in more detail later.
```

Comment:
`for example if Bob wants to verify Alice's data then he would input Alice's MSPID.` -> `for example if Bob wants to verify data in Alice's collection, then he would input Alice's MSPID.`

Reply:
```
For example, if Alice stores some private data `{"myPrivateValue":"50"}` in her Private Data Collection, identifiable by the MSPID `AliceOrg1MSP`, which only she has access to but wants to prove to Bob that it exists, Bob would use the verify transaction to confirm whether Alice is telling the truth or not. 
```

->

```
For example, if Alice stores some private data `{"myPrivateValue":"50"}` in her Private Data Collection, Bob could use the verify transaction to confirm whether Alice is telling the truth or not. 
```